sdk/src/dlob/DLOB.ts

```diff
@@ -131,6 +133,72 @@ export class DLOB {
 		return true;
 	}
 
+	public handleOrderRecord(record: OrderRecord): void {
+		this.insertOrder(record.order, record.user);
+	}
+
+	public handleOrderActionRecord(record: OrderActionRecord): void {
+		if (isVariant(record.action, 'place')) {
```

Comment:
should you also return early if the action == `expire`?

Reply:
yup